docs/howtos/Client Generation/05union.mdx

```diff
@@ -37,11 +37,11 @@ op completions(@body input: CompletionInput): CompletionResult;
 <TabItem value="python" label="Python">
 
 ```python
-def analyze(*, mode: Literal["strict", "lenient"], image: bytes) -> AnalyzeResult:
+def analyze(image: bytes, *, mode: Literal["strict", "lenient"]) -> AnalyzeResult:
   ...
 
 class CompletionInput:
-  input: str | List[str] = rest_field(readonly=True)
+  input: Union[str, List[str]] = rest_field(readonly=True)
```

Comment:
Introduced in 3.10: https://docs.python.org/3/library/stdtypes.html#types-union and 3.8/3.9 doesn't support yet.